AssetRipper.Library/Ripper.cs

```diff
@@ -234,6 +236,36 @@ public void ResetData()
 				return asset => types.Any(t => asset.GetType().IsAssignableTo(t));
 			}
 		}
+		private static Func<IUnityObjectBase, bool> GetFilter(String filespec)
+		{
+			if (filespec == null)
+			{
+				return LibraryConfiguration.DefaultFilter;
+			}
+			else if (filespec == "")
+			{
+				return asset => false;
+			}
+			else
+			{
+				Regex fileregex = new Regex(
+					"^"+filespec.Replace(".","\\.").Replace("*","[^/]*").Replace("[^/]*[^/]*",".*").Replace("?",".")+"$",
```

Comment:
`Microsoft.Extensions.FileSystemGlobbing` provides a cross platform way to do glob matching instead of my hacky non-portable regex solution here, but I couldn't get it to behave so I put a pin in it until I get some feedback on whether this PR is generally desirable.